analyzer-comments/ruby/leap/std_lib.md

```diff
@@ -0,0 +1,3 @@
+# std lib
+
+Avoid using `Date.leap?` from the Standard Library
```

Comment:
The header seems to not be capitalized, and there is no period at the end of this sentence.

Reply:
I added a period. The convention seems to be to not capitalize headers so I will leave the header as it is.